cmd/git-sync/main.go

```diff
@@ -1075,7 +1101,6 @@ func runCommand(ctx context.Context, cwd, command string, args ...string) (strin
 
 func runCommandWithStdin(ctx context.Context, cwd, stdin, command string, args ...string) (string, error) {
 	cmdStr := cmdForLog(command, args...)
-	log.V(5).Info("running command", "cwd", cwd, "cmd", cmdStr)
```

Comment:
these logs are useful, please don't remove them :)

Reply:
this is actually, to be able to test runCommand , test case fails as "log" is not inject during tests ?

Reply:
ok, so we probably need to initialize `log` as `&customLogger{logr.Discard(), "", ""}`

Reply:
created cmd package which is responsible of running commands and it keeps logger as struct field.
reverted logs back.